src/utils.ts

```diff
@@ -639,8 +639,12 @@ export namespace Utils {
     const now = Math.floor(Date.now() / 1000);
     const { minTime, maxTime } = transaction.timeBounds;
 
+    // give a small grace period for the transaction time to account for clock drift
+    const gracePeriod = 60 * 5;
```

Comment:
Could you make this a parameter of the function, and pass the value in from the SEP-10 read function? This function is only used by the SEP-10 code today, but it is rather generic and the 5 minute time is appropriate for SEP-10 but may not be appropriate for other use cases.